Packs/Zoom/Integrations/Zoom/Zoom.py

```diff
@@ -1624,43 +2084,366 @@ def zoom_list_messages_command(client, **args) -> CommandResults:
+
+        mirror = {
+            'channel_jid': channel_jid,
+            'channel_id': channel_id,
+            'channel_name': channel_name,
+            'investigation_id': investigation.get('id'),
+            'mirror_type': type,
+            'mirror_direction': direction,
+            'auto_close': bool(autoclose),
+            'mirrored': True
+        }
+    else:
+        mirror = mirrors.pop(mirrors.index(current_mirror[0]))
+        channel_jid = mirror['channel_jid']
+        channel_id = mirror['channel_id']
+        if type:
+            mirror['mirror_type'] = type
+        if autoclose:
+            mirror['auto_close'] = autoclose
+        if direction:
+            mirror['mirror_direction'] = direction
+        if channel_name:
+            # update channel name
+            result = zoom_update_channel_command(client=client, user_id=admin_user_id,
+                                                 channel_name=channel_name, channel_id=channel_id)
+            mirror['channel_name'] = channel_name
+        channel_name = mirror['channel_name']
+        mirror['mirrored'] = True
+    demisto.mirrorInvestigation(investigation_id, f'{type}:{direction}', autoclose)
+
+    mirrors.append(mirror)
+    set_to_integration_context_with_retries({'mirrors': mirrors}, OBJECTS_TO_KEYS, SYNC_CONTEXT)
+
+    if send_first_message:
+        server_links = demisto.demistoUrls()
+        server_link = server_links.get('server')
+        message = (f'This channel was created to mirror incident {investigation_id}.'
+                   f' \n View it on: {server_link}#/WarRoom/{investigation_id}')
+        url_suffix = '/im/chat/messages'
+        json_data_all = {
+            "content": {
+                "body": [
+                    {
+                        "type": "message",
+                        "text": message
+                    }
+                ]
+            },
+            "to_jid": channel_jid,
+            "robot_jid": client.botJid,
+            "account_id": client.account_id
+        }
+        client.zoom_send_notification(url_suffix, json_data_all)
+    # if kick_admin:
+    #     demisto.debug("kick-admin:")
+    #     url_suffix = f'/chat/users/{admin_user_id}/channels/{channel_id}/members/{admin_user_id}'
+    #     res = client.zoom_remove_from_channel(url_suffix)
+    #     demisto.debug(f"res: {res}")
+    return CommandResults(
+        readable_output=f'Investigation mirrored successfully, channel:{channel_name},{channel_jid}'
+    )
+
+
+def find_mirror_by_investigation() -> dict:
+    """
+    Finds a mirrored channel by the mirrored investigation
+
+    Returns:
+        The mirror object
+    """
+    mirror: dict = {}
+    investigation = demisto.investigation()
+    demisto.debug(f"investigation: {investigation}")
+    if investigation:
+        integration_context = get_integration_context(SYNC_CONTEXT)
+        demisto.debug(f"integration_context: {integration_context}")
```

Comment:
Can it be very large?
In that case, maybe it shouldn't be logged?
```suggestion
        demisto.debug(f"{integration_context=}")
```